### ISSUE TITLE
Use stdint.i instead of stdint.h in auditswig.i

### ISSUE THREAD

**Issue by bigon** [linux-audit/audit-userspace#96]
Hello,

In https://github.com/linux-audit/audit-userspace/blob/master/bindings/swig/src/auditswig.i#L44 we are using stdint.h.

Apparently this is causing issues, (see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=927946) that makes the python binding unusable.

Using stdint.**i** fixes the issues:

```C
-#include <stdint.h>
+%include <stdint.i>
```

**Comment by stevegrubb** [comment]
Fixed in commit c46c8f2. Thanks for reporting the issue.

**Comment by EccoTheFlintstone** [comment]
Bug seem to still be present (with latest version 2.8.5  , version 2.8.5-2ubuntu-1 on ubuntu 19.10)
proposed fix by @bigon was
`%include <stdint.i>`

what was commited is
`#include <stdint.i>`

Note the difference between `%include` and `#include `which is a big difference with SWIG
From my tests with : 
```
1 - #include <stdint.h>
2 - #include <stdint.i>
3 - %include <stdint.i>
```

wrapper generated by 1 and 2 are the same
wrapper generated by 3 uses a different SWIG function to parse uint32_t and should be ok


quick test to see if it works : 
```
import audit
fd = audit.audit_open()
audit.audit_set_enabled(fd, 1)
```
Currently, this is the error we get :
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: in method 'audit_set_enabled', argument 2 of type 'uint32_t'
>>>
```
